### ISSUE TITLE
category axis labels do not align with columns?

### ISSUE THREAD

**Issue by morrow95** [amcharts/amcharts5#155]
Any ideas what would cause this?  x-axis is category axis.  yaxis is duration axis.  The x-axis labels do not align at all with the columns.  I have other charts which have a category axis as the x-axis and this is not an issue - they are centered with their column and 'under' the cursor when hovering over the column.  In my screenshot I zoomed in to show it better, but it does not matter if it is zoomed in or not - the labels are not centered with their columns.

![Untitled-1](https://user-images.githubusercontent.com/626244/146490740-a651ed1a-76ac-4cfa-86f7-5d5ff30c5a4b.jpg)

**Comment by martynasma** [comment]
It's by design. When grid is placed not for each base interval, the label is moved to the beginning of the cell so it is not confuses users. It would be misleading to show 8PM in the middle of the 8-9PM range.

You can configure it as you like though:
https://www.amcharts.com/docs/v5/charts/xy-chart/axes/#Location_of_axis_elements

**Comment by morrow95** [comment]
@martynasma  That is the thing though... I provide a value in my data for 12pm through 11pm... even if the value is 0.  Not to mention the x-axis is a category axis so there is no base interval for it.

**Comment by martynasma** [comment]
This works the same way for CategoryAxis. If grid lines do not fit comfortably, they get get skipped. When grid lines are skipped, labels are moved to the left.

The tutorial I linked to explains how you can configure this behavior to fit your needs.

**Comment by morrow95** [comment]
I've tried all sorts of combinations from that page.  I can get the grid lines to match up vertically with the columns, but the x-axis labels never line up with the columns/cursor.  I made a codepen of the same data at [https://codepen.io/scubapongo/pen/GRMmwKj](https://codepen.io/scubapongo/pen/GRMmwKj)

It just looks really odd to me this way.  If you don't hover over a column to see the cursor/tooltip you think the columns are at a different time (hour in this case) then what they are because the labels don't line up vertically with them.  I get that not all the labels will always show, I don't get why the ones that do aren't lined up with their respective column (since these are categories and just string names - not a date/time).
